app/_src/gateway/plugin-development/wasm.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Create a Proxy-Wasm filter
+book: plugin_dev
+chapter: 11
+---
+
+## Your first Proxy-Wasm filter
+
+Filters can be written as a compilation target in almost any language that supports WebAssembly, but the easiest way to get started is by
+using a language which has an existing [Proxy-Wasm](https://github.com/proxy-wasm/spec)
+SDK, such as [Go](https://github.com/tetratelabs/proxy-wasm-go-sdk/)
+or [Rust](https://github.com/proxy-wasm/proxy-wasm-rust-sdk/).
+
+## Coding, building and deploying a Proxy-Wasm filter
+
+Kong has made Go and Rust Proxy-Wasm filter templates available. They can be used
+to start writing your own filters, or to build as they are and get your first `hello
+world`-type filter:
```

Comment:
I think it should also be specified that other Proxy-Wasm SDKs are "theoretically supported but not fully tested by Kong Inc. yet".